tracer/build/_build/BuildVariables.cs

```diff
@@ -15,25 +15,34 @@ public static void AddContinuousProfilerEnvironmentVariables(this Dictionary<str
         envVars.AddTracerEnvironmentVariables(tracerHomeDirectory);
     }
 
-    public static void AddTracerEnvironmentVariables(this Dictionary<string, string> envVars, AbsolutePath tracerHomeDirectory)
+    public static void AddTracerEnvironmentVariables(this Dictionary<string, string> envVars, AbsolutePath monitoringHomeDirectory)
     {
         envVars.Add("COR_ENABLE_PROFILING", "1");
         envVars.Add("COR_PROFILER", "{846F5F1C-F9AE-4B07-969E-05C26BC060D8}");
-        envVars.Add("COR_PROFILER_PATH_32", tracerHomeDirectory / "win-x86" / "Datadog.Trace.ClrProfiler.Native.dll");
-        envVars.Add("COR_PROFILER_PATH_64", tracerHomeDirectory / "win-x64" / "Datadog.Trace.ClrProfiler.Native.dll");
+
+        envVars.Add("COR_PROFILER_PATH_32", monitoringHomeDirectory / "Datadog.AutoInstrumentation.NativeLoader.x86.dll");
+        envVars.Add("COR_PROFILER_PATH_64", monitoringHomeDirectory / "Datadog.AutoInstrumentation.NativeLoader.x64.dll");
+
+        if (EnvironmentInfo.IsWin)
+        {
+            envVars.Add("DD_DOTNET_TRACER_HOME", monitoringHomeDirectory / "tracer");
```

Comment:
I take your word for it, but I feel like it would be easier to fix the Windows layouts first, as we're going to have to change all this _back again_ otherwise?

Reply:
It feels like the work to change the layout for windows would be a bit difficult (checking AAS, reviewing the MSI installer project, dd-tool...)
But I get your point, this is just a workaround